test/util/display_SYSINFO_settings.py

```diff
@@ -1,31 +1,28 @@
-# Copyright 2016, EMC, Inc.
+'''
+Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
feel free to do a mv of the files.  mv dispplay_SYSINFO_settings.py display_sysinfo_settings.py and same for display_node_FIRMWARE_versions.py.
That keeps the log history and easy fix.